ui/perfherder/alerts/AlertsViewControls.jsx

```diff
@@ -114,7 +114,8 @@ export default class AlertsViewControls extends React.Component {
           dropdownCol
         />
         {pageNums
-          ? pageNums.length > 1 && (
+          ? pageNums.length > 0 &&
+            count !== 1 && (
```

Comment:
This is working correctly, I wouldn't change it.

Reply:
@alexandru-io do you mean that the pages navigation bar at the top should disappear when a user reaches the last page? If we still have a working bar at the bottom, the alerts view won't get stuck, but I thought it would make sense to have a working navigation at the top as well.

Reply:
Oh, my bad, I thought it was something else. Great catch, then

Reply:
This solution works.
But I would do it slightly different.
Just before the return, I would add a 
`const hasMorePages = pageNums.length > 0 && count !== 1;`
And the check would be:
```
pageNums
          ? hasMorePages && (
              <Row .........
```
This way the code is more explicit and the `pageNums.length > 0 && count !== 1` condition have a name.

Reply:
@sviho could you follow Octavian's suggestion. We're very close to finishing this PR.